steps/appaloosa/1.0.0/step.yml

```diff
@@ -0,0 +1,167 @@
+title: Publish to Appaloosa Store
+summary: This step allows you to publish your application to your private app store
+  on Appaloosa Store
+description: |
+  Bitrise Step to Deploy an iOS IPA to Appaloosa-Store.
+
+  Required: an Xcode Archive Step should run before this Step!
+
+  This step requires an Appaloosa-Store account or an email address to create your account
+website: https://www.appaloosa-store.com
+source_code_url: ""
+support_url: ""
```

Comment:
also this one - it can be the `/issues` link on GitHub: https://github.com/appaloosa-store/bitrise_appalossa/issues